elements/media-quote/demo/index.html

```diff
@@ -19,8 +19,13 @@
       <h3>Basic media-quote demo</h3>
       <demo-snippet>
         <template>
-          <media-quote>
-            This is media-quote
+          <media-quote 
+            media-src="https://sc-api.psu.edu/s3/files/styles/4_3_1500w/public/2023/04/old-main-blue-sky.jpg?h=08b866d1&itok=ICYTxRXL"
+            media-alt="Old Main in front of a blue sky"
+            author="John Doe"
+            author-profession="Professor at Penn State University"
+            caption="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ultrices sagittis orci a scelerisque purus semper eget duis at. Ipsum dolor sit amet consectetur adipiscing elit. Eu non diam phasellus vestibulum lorem. Ullamcorper a lacus vestibulum sed arcu non odio. Auctor eu augue ut lectus. Ut sem nulla pharetra diam sit amet nisl suscipit adipiscing. Arcu odio ut sem nulla pharetra diam. Proin libero nunc consequat interdum varius sit. Eros donec ac odio tempor orci. Tortor pretium viverra suspendisse potenti nullam ac tortor vitae purus. Aenean euismod elementum nisi quis eleifend quam. Odio euismod lacinia at quis risus sed vulputate odio ut. Quam id leo in vitae turpis massa sed elementum. Sed odio morbi quis commodo odio. Condimentum vitae sapien pellentesque habitant morbi tristique senectus et netus. Donec ultrices tincidunt arcu non sodales neque sodales ut etiam. Feugiat vivamus at augue eget arcu dictum. Vel eros donec ac odio tempor orci dapibus. Sit amet consectetur adipiscing elit.">
```

Comment:
Add support for caption and author to be loaded in via a slot. We Always get requests for being able to bold / <i> text that's in blocks of text and this isn't possible for this currently. see named slots like `<slot name="caption">${this.caption}</slot>` as a way of supporting both.

Reply:
Sounds good. In the HTML, does that essentially work like this:

<media-query>
  <quote>"This is a quote</quote>
  <author>John Doe</author>
  <author-detail>Professor at Penn State University</author-detail>
  <caption>This is a caption</caption>
</media-query>

and the <slot name="caption"></slot> gets the data from the caption tag or is there a different way that would work?